From 7b74d91b8808f10cc99a2197c70aa9b11a2fc041 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Tue, 27 Jun 2017 12:45:33 -0700
Subject: [PATCH 2/2] Build nfc_nci.nqx.default with BOARD_VNDK_VERSION

Bug: 37342627
Test: BOARD_VNDK_VERSION=current m -j64 nfc_nci.nqx.default.vendor
Change-Id: Ic700f1769eefe880deeae901b658685e00772206
---
 halimpl/pn54x/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/halimpl/pn54x/Android.mk b/halimpl/pn54x/Android.mk
index 1e8fc54..4a6a117 100644
--- a/halimpl/pn54x/Android.mk
+++ b/halimpl/pn54x/Android.mk
@@ -125,6 +125,8 @@ endif
 LOCAL_CFLAGS += $(D_CFLAGS)
 #LOCAL_CFLAGS += -DFELICA_CLT_ENABLE
 #-DNXP_PN547C1_DOWNLOAD
+
+LOCAL_HEADER_LIBRARIES += libhardware_headers
 include $(BUILD_SHARED_LIBRARY)
 
 include $(CLEAR_VARS)
-- 
2.17.1

